builds/azure-pipelines/build-release.yml

```diff
@@ -28,7 +28,7 @@ variables:
   versionPatch: $[counter(variables['versionMajorMinor'], 0)] # This will reset when we bump minor version
   binariesVersion: '$(versionMajor).$(versionMinor).$(versionPatch)'
   nugetVersion: $[replace(replace('True', eq('${{ parameters.ReleaseType }}', 'full'), '$(binariesVersion)'),'True','$(binariesVersion)-preview')] # This will set the preview tag to the version based on the ReleaseType parameter.
-  supportedVersion: '$(versionMajor).*-*'
+  supportedVersion: $[replace(replace('True', eq('${{ parameters.ReleaseType }}', 'full'), '$(versionMajor).*'),'True','$(versionMajor).*-*')] # This will set the expression to pull the preview vs full version based on the ReleaseType parameter.
```

Comment:
It would be good to verify that 3.* won't also pick up preview versions (e.g. 3.0.143-preview)

It's not something we'll be running into currently, but in the future I can see us doing preview release of the same major version and we don't want this to cause problems then.

Reply:
So we'll be ahead but in the minor versions, okay will look into how we can add that check then. Thanks!